filecoin-proofs/src/api.rs

```diff
@@ -481,3 +589,121 @@ fn pad_safe_fr(unpadded: &FrSafe) -> Fr32Ary {
     res[0..31].copy_from_slice(unpadded);
     res
 }
+
+#[test]
+fn test_generate_piece_commitment() -> Result<(), failure::Error> {
+    fn blah(data: &[u8]) -> Result<Commitment, failure::Error> {
```

Comment:
TODO: Come up with a better name than `blah`.

Reply:
TODO: Move this function outside of the `test_generate_piece_commitment` function body.